client/src/utils/sendNotification.js

```diff
@@ -0,0 +1,24 @@
+import toast from 'cogo-toast';
+
+const option = {
+  hideAfter: 1.5
+};
+
+export const showSuccessNotif = (message) => toast.success(message, option);
+
+export const showLoadingNotif = (message) => toast.loading(message, option);
```

Comment:
i don't see these functions used anywhere else except this folder, do we want to export it?

Reply:
We need the `showSuccessNotif` when user favourites a course :)